### DISCUSSION TITLE
git push issues

### DISCUSSION THREAD

**Discussion by erikamoji** [swellstores/community#178] [Help] [answered]
> # Store-ID: amore-creations

We recently have had trouble publishing changes to the storefront using the editor. After attempting to publish, here is the log:
```
​Deploying branch: production
Preparing environment...
```
Pushing the changes via `git push` publishes the changes but results in errors pulling from the backend (see photo below for an example)
![Screenshot 2023-03-14 113804](https://user-images.githubusercontent.com/33100208/225056031-20dd6c4a-3586-4c5f-a3f6-3f8a50eb2ffb.png)
![Screenshot 2023-03-14 113835](https://user-images.githubusercontent.com/33100208/225056051-b3889f64-6d2b-4d5b-aa2e-40864b10c0fb.png)

We are happy to continue pushing updates via git instead of the editor, so can someone help us through how to fix the problems with the photos above?

---

### Also discussed in https://github.com/orgs/swellstores/discussions/77

<div type='discussions-op-text'>

<sup>Originally posted by **fmanaa** January  8, 2023</sup>
I am on the community version... When I make changes to the theme and publish from the GUI (Storefront -> Theme -> Edit Theme -> Publish), the changes only affect the https://XYZ--dev.swell.store/ version and not the https://XYZ.swell.store/ or the https://XYZ.apps.swellstores.com/. How do I propagate the change all the way through?
</div>

**Comment by Strongeyed** [comment]
Similar issues here on two stores run through Swell, haven't been able to publish for the last 3 weeks, publishing takes only seconds (normally minutes) to go from preparing environment to completed. Only error message I could find is attached in the screenshot below but I'm not sure if this relates to the issue. Keen to hear if you had any luck resolving this issue?

![Screenshot 2023-03-17 at 11 17 39 AM](https://user-images.githubusercontent.com/128098520/225764943-497c066a-ba62-4b8d-8e65-b265d50e715e.png)

**Reply by erikamoji** [reply]
None, and unfortunately, we don't have any errors from our logs either.

**Reply by Strongeyed** [reply]
Must be something on the server side, not sure how to proceed now as it's been raised as a ticket 3 weeks ago, patience I suppose

**Comment by rodrigoddalmeida** [comment, marked as answer]
@erikamoji @Robin-DG could you update your storefronts (Origin) to the latest version, then publish? That should take care of the error you see. You can do it through the Admin Dashboard UI or by rebasing/merging from Origin's public repository if you have custom code.

**Reply by logeshswell** [reply]
@erikamoji It looks like the translation issue in your storefront is working fine now, after our recent release.
